crates/cranelift/src/debug/transform/simulate.rs

```diff
@@ -345,13 +343,7 @@ pub fn generate_simulated_dwarf(
         (unit, root_id, file_id)
     };
 
-    let mut module_wasm_types = PrimaryMap::new();
-    for (module, memory_offset) in compilation.module_memory_offsets.iter() {
-        let wasm_types = add_wasm_types(unit, root_id, out_strings, memory_offset);
-        let i = module_wasm_types.push(wasm_types);
-        assert_eq!(i, module);
-    }
-
+    let wasm_types = add_wasm_types(unit, root_id, out_strings);
```

Comment:
Since all of the remaining `wasm_types` are module-invariant, there is no longer a need for them to be per-module.